puzzle/SliderPuzzleUI.java

```diff
@@ -147,22 +153,22 @@ private static void DoUI() {
 			}
 		} while (size != 3 && size != 5);
 		scan.close();
-		
```

Comment:
hahahahah I sure am @domfarolino. I noticed it in the diff and but didn't care enough to fix it or change my atom config (I just installed atom so I haven't done much to configure it yet)